crates/sui-framework/packages/sui-system/tests/rewards_distribution_tests.move

```diff
@@ -628,4 +630,59 @@ module sui_system::rewards_distribution_tests {
         test_scenario::return_shared(sui_system);
         test.end();
     }
+
+    #[test]
+    fun test_process_pending_stake_withdraw_no_underflow_in_safe_mode() {
```

Comment:
I don't see a single assert with `pool_token_balance > pending_pool_token_withdraw`, and without it the change is not justified, no?

Reply:
Yeah, sorry, was already wrapping up for the day when I left this comment. I think that explicit comparison would make the PR more clear.